docker-compose.migrate.yaml

```diff
@@ -17,7 +17,7 @@ services:
   database-up:
     image: liquibase/liquibase:3.10.x
     environment:
-      << : [*common-postgres, *common-migration]
+      <<: [*common-migration, *common-postgres]
```

Comment:
What's the reason for reversing these?  They get applied in reverse order so thought they had to be as was

Reply:
Though as there's no conflict guess it doesn't matter